deploy/ssh-setup.sh

```diff
@@ -33,6 +33,6 @@ else
     if ssh-add ~/firesim.pem; then
         echo "success: firesim.pem added to ssh-agent"
     else
-        echo "FAIL: ERROR adding ~/firesim.pem to ssh-agent. does it exist?"
+        echo "FAIL: ERROR adding ~/firesim.pem to ssh-agent. If on AWS F1, does it exist?"
```

Comment:
I think that regardless of whether you are on F1 or a local cluster or some other cloud provider, you are going to be using Fabric to SSH to other hosts and you will want key-based auth to work.  I don't think we should make this message F1 specific but we could clean this up in a follow-on.